autoscheduler/frontend/src/components/LandingPage/HelpText.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {
+  Paper, Typography, Grid, Box,
+} from '@material-ui/core';
+
+const HelpText: React.SFC = () => (
+  <Grid
+    container
+    direction="row"
+    justify="space-around"
+    alignItems="center"
+    style={{ marginTop: '5%', marginBottom: '2%' }}
```

Comment:
Don't think it's as necessary as it is for `SelectTerm.tsx`, but maybe put the necessary css for this into a separate file.